docs/style.css

```diff
@@ -0,0 +1,76 @@
+html {
+  scroll-behavior: smooth;
+}
+
+#js-producthunt {
+  margin-top: 1rem;
+}
+
+#installation {
+  overflow: hidden;
+}
+#installation header > h3 {
+  margin: 1rem 2.5rem 1rem 0;
+}
+/* Make the ✔ / ❌ Emoji appear next to the button */
+#installation header button {
```

Comment:
Sure happy, to switch these!
I tried to avoid `.class` selectors where ever possible to stay true to the "no classes"-idea of water.css (and the ID is needed for semantics/scroll anchor anyhow), but yeah, adding one will definitely make this more readable & maintainable.

Reply:
Using classes now. :)